bin/cron/push_latest_rds_backup_to_secondary_account

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+
+Dir.chdir(File.expand_path('../../../dashboard', __FILE__))
+require_relative '../../dashboard/config/environment'
+require 'aws-sdk'
+require 'cdo/hip_chat'
+
+# This script pushes backup snapshots of our production database into a
+# write-only backup account, with the following steps:
+#
+# 1) Copy the latest automated snapshot to a temporary manual one
+# 2) Wait until ready, and share the manual snapshot with the backup acct
+# 3) On the backup account, copy the shared snapshot to a manual one
+# 4) Wait until ready, and finally go back to the main account and delete the temp snapshot
+#
+# Credential sets must be set up with the names 'default' and 'backup' in the aws config directory,
+# which requires setting appropriate secrets in our Chef config (which will come through via crontab.erb)
+#
+
+def get_aws_account_id(credentials)
+  # based on https://gist.github.com/thomaswitt/3952303, updated for new Aws SDK
+  begin
+    iam = Aws::IAM::Client.new(credentials: credentials)
+    iam = iam.get_user
+    id, user = iam[:user][:arn].match('^arn:aws:iam::([0-9]{12}):(.*)$').captures
+  rescue Aws::IAM::Errors::AccessDenied
+    # This happens because in our case, we have some accounts that are so locked down
+    # that they don't even have IAM get_user permissions. But AWS still spits the
+    # user back in the error message. This is gross, but there doesn't seem to be
+    # any other way.
+    result = $!
+    id, user = result.to_s.match('^User: arn:aws:iam::([0-9]{12}):(.*) is not auth.*$').captures
+  end
+  # rubocop:disable Performance/RedundantMatch
+  raise 'incorrect account id' unless id.match(/^[0-9]{12}$/)
+  # rubocop:enable Performance/RedundantMatch
+
+  [id, user]
+end
+
+def wait_for_snapshot_copy(snapshot)
+  waited = 0
+  until snapshot.status == 'available'
+    snapshot.reload
+    puts "Snapshot status for #{snapshot.snapshot_id}: " + snapshot.status + ' - ' + Time.now.to_s
```

Comment:
you could add `#{snapshot.status} - #{Time.now}` to the existing interpolated string rather than concatenate these values at the end